### PR TITLE
Support Sneegleeb and can of mixed everything

### LINKED ISSUE #1533: Offhand equips during free fights
Had Shield of the Skeleton Lord equipped for the entirety of my free fight section, but wouldn't something like sneebgleeb or can of mixed everything be better in this situation? Or even latte? It was fishing for marginal fam drops so I do not believe the famwt was contributing to anything.

### PR DESCRIPTION
**Pull request by horrible-little-slime**
Resolves #1533

### CODE REVIEW

File: packages/garbo/src/outfit/dropsgear.ts [hunk 1]
```diff
@@ -364,3 +365,22 @@ function powerGlove(): Map<Item, number> {
     ],
   ]);
 }
+
+const POSSIBLE_SNEEGLEEB_DROPS = Item.all().filter(
+  (i) =>
+    i.tradeable && i.discardable && (i.inebriety || i.fullness || i.potion),
+);
+function sneegleebs(): Map<Item, number> {
+  const dropValue =
```
Comment (s-k-z) [inline packages/garbo/src/outfit/dropsgear.ts:374]: May be worth caching this value, it's currently over 2000 items being averaged every time and gets checked every combat.

File: packages/garbo/src/outfit/dropsgear.ts [hunk 2]
```diff
@@ -364,3 +365,28 @@ function powerGlove(): Map<Item, number> {
...
+      Math.min(garboValue(item), MAX_SNEEGLEEB_PRICE),
+    ) / POSSIBLE_SNEEGLEEB_DROPS.length;
+  return new Map<Item, number>(
+    (
+      [
+        [$item`KoL Con 13 snowglobe`, sneegleebDropValue * SNEEGLEEB_DROP_RATE],
+        [
+          $item`can of mixed everything`,
+          (sneegleebDropValue * SNEEGLEEB_DROP_RATE) / 2,
+        ],
```
Comment (s-k-z) [inline packages/garbo/src/outfit/dropsgear.ts:388]:
```suggestion
let sneegleebBonus: number;
const SNEEGLEEB_DROP_RATE = 0.13;
const MAX_SNEEGLEEB_PRICE = 100_000; // arbitrary, to help avoid outliers
function sneegleebs(): Map<Item, number> {
  sneegleebBonus ??=
    sum(POSSIBLE_SNEEGLEEB_DROPS, (item) =>
      Math.min(garboValue(item), MAX_SNEEGLEEB_PRICE),
    ) / POSSIBLE_SNEEGLEEB_DROPS.length * SNEEGLEEB_DROP_RATE;
  return new Map<Item, number>(
    (
      [
        [$item`KoL Con 13 snowglobe`, sneegleebBonus],
        [$item`can of mixed everything`, sneegleebBonus / 2],
```
Could make it a little more concise.

Comment (horrible-little-slime) [reply packages/garbo/src/outfit/dropsgear.ts]: I had considered this, but hadn't thought of the right variable name. 

Great choice!

**Comment by s-k-z** [comment]
Lowercase acronyms are evil

**Pull request merged**